crates/control/src/foot_bumper_filter.rs

```diff
@@ -133,24 +133,31 @@ impl FootBumperFilter {
             .push_back(self.right_pressed_last_cycle);
         self.right_detection_buffer.pop_front();
 
-        let obstacle_detected_on_left = self.left_count >= *context.activations_needed;
-        let obstacle_detected_on_right = self.right_count >= *context.activations_needed;
-
         self.check_for_bumper_errors(&context);
 
         if *fall_state != FallState::Upright {
             return Ok(Default::default());
         }
 
-        let obstacle_angle = match (
-            obstacle_detected_on_left && self.left_in_use,
-            obstacle_detected_on_right && self.right_in_use,
-        ) {
-            (true, true) => 0.0,
-            (true, false) => *context.sensor_angle,
-            (false, true) => -context.sensor_angle,
-            _ => return Ok(Default::default()),
-        };
+        let obstacle_detected_on_left =
+            (self.left_count >= *context.activations_needed) && self.left_in_use;
+        let obstacle_detected_on_right =
+            (self.right_count >= *context.activations_needed) && self.right_in_use;
+        let obstacle_detected_on_middle = ((self.right_count + self.left_count)
+            >= *context.activations_needed)
+            && (self.left_in_use || self.right_in_use);
```

Comment:
Shouldn't this be
```suggestion
            && (self.left_in_use && self.right_in_use);
```

???

Reply:
Done.